src/jllvm/debuginfo/TrivialDebugInfoBuilder.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2023 The JLLVM Contributors.
+//
+// This file is part of JLLVM.
+//
+// JLLVM is free software; you can redistribute it and/or modify it under  the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 3, or (at your option) any later version.
+//
+// JLLVM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+// of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along with JLLVM; see the file LICENSE.txt.  If not
+// see <http://www.gnu.org/licenses/>.
+
+#include "TrivialDebugInfoBuilder.hpp"
+
+#include <llvm/IR/Function.h>
+
+jllvm::TrivialDebugInfoBuilder::TrivialDebugInfoBuilder(llvm::Function* function)
+    : m_debugBuilder(*function->getParent())
+{
+    llvm::DIFile* file = m_debugBuilder.createFile(".", ".");
+    unsigned runtimeVersion = 1;
```

Comment:
Is this a magic number?

Reply:
Yesn't. I think this number is used by some debuggers to communicate version numbers. I read about this being the case for Objective-C. But besides that, everything greater than zero is apparently okay.

Reply:
Could you make it a constant then?